Orange/widgets/data/owfile.py

```diff
@@ -93,21 +93,39 @@ def __init__(self):
         box.setSizePolicy(Policy.MinimumExpanding, Policy.Fixed)
         self.file_combo.setSizePolicy(Policy.MinimumExpanding, Policy.Fixed)
         self.file_combo.activated[int].connect(self.select_file)
-        box.layout().addWidget(self.file_combo)
-        button = gui.button(
-            box, self, '...', callback=self.browse_file, autoDefault=False)
-        button.setIcon(self.style().standardIcon(QtGui.QStyle.SP_DirOpenIcon))
-        button.setSizePolicy(Policy.Maximum, Policy.Fixed)
-        button = gui.button(
-            box, self, "Reload", callback=self.reload, autoDefault=False)
-        button.setIcon(self.style().standardIcon(QtGui.QStyle.SP_BrowserReload))
-        button.setSizePolicy(Policy.Fixed, Policy.Fixed)
-        layout.addWidget(box, 0, 1,  QtCore.Qt.AlignVCenter)
+        layout.addWidget(self.file_combo, 0, 1)
```

Comment:
If you do it like this, `box` is not inserted into the layout. Moreover, the sheet combo is not right-aligned to the file combo due to some additional spacing to the right (I've not idea where it comes from). Inserting the file combo into the box and box into the layout will align the two combos. Thus, replace this line with

        box.layout().addWidget(self.file_combo)
        layout.addWidget(box, 0, 1)